integration/java/util/io/opencensus/interop/util/TestUtils.java

```diff
@@ -144,13 +152,14 @@ public static boolean verifyResponse(
       try {
         TagContext actualTagContext =
             serializer.fromByteArray(response.getTagsBlob().toByteArray());
-        if (!actualTagContext.equals(expectedTagContext)) {
+        Set<Tag> actualTags = Sets.<Tag>newHashSet(InternalUtils.getTags(actualTagContext));
+        actualTags.remove(Tag.create(METHOD_KEY, METHOD_VALUE));
```

Comment:
I think there should be a comment or TODO explaining why the method tag is ignored.

Reply:
SG, done.